integration-tests/jpa-repository/src/main/resources/reflection-config.json

```diff
@@ -0,0 +1,11 @@
+[
+  {
+    "name" : "org.jberet.runtime.JobInstanceImpl",
```

Comment:
This should be registered in the extension

Reply:
I do not agree,
You do the same in the integration test for jdbc (so why should this be wrong for jpa) ? : 
https://github.com/quarkiverse/quarkus-jberet/blob/417e66132131aede3e90f8205b843ee9d78d98fe/integration-tests/jdbc-repository/src/main/resources/reflection-config.json#L3

This is a requirement for the integration test, because you created a rest resources returning org.jberet.runtime.JobInstanceImpl, and so jsonb needs to properly deserialize it (so we need to disable threeshaking).

This is not required in general

Reply:
Yes, I guess this was my bad. I was probably trying out the things required for native and then forgot to register them.

Ideally, we prefer to register such classes with `ReflectiveClassBuildItem` in the Processor, so the user / consumer does not have to guess which classes have to be registered for reflection via file to run in the native image.

My recommendation is to just register this and remove both reflection files.

Reply:
Done !